worldengine/generation.py

```diff
@@ -176,31 +176,45 @@ def _around(x, y, width, height):
 def generate_world(w, step):
     if isinstance(step, str):
         step = Step.get_by_name(step)
-    seed = w.seed
 
     if not step.include_precipitations:
         return w
 
+    # Prepare sufficient seeds for the different steps of the generation
+    rng = numpy.random.RandomState(w.seed)  # create a fresh RNG in case the global RNG is compromised (i.e. has been queried an indefinite amount of times before generate_world() was called)
+    sub_seeds = rng.randint(0, 4294967295, size=100)  # sys.maxsize didn't quite work
```

Comment:
I don't like the use of this fixed number at all. Does somebody have a better idea? sys.maxint (or whatever it is called) is not available for Python 3; sys.maxsize returns a number that is not compatible with numpy (which takes a uint32).